slices/git.yaml

```diff
@@ -6,33 +6,36 @@ essential:
 slices:
   bins:
     # basic functionality including:
-    # - init
-    # - config
     # - add
-    # - commit
-    # - status
-    # - log
-    # - checkout
     # - blame
+    # - branch
+    # - checkout
+    # - commit
+    # - config
     # - diff
+    # - fetch
+    # - init
+    # - log
     # - merge
+    # - rev-list
+    # - status
     essential:
       - libc6_libs
       - libpcre2-8-0_libs
       - zlib1g_libs
     contents:
-      /usr/bin/git: {symlink: /usr/lib/git-core/git}
+      /usr/bin/git: { symlink: /usr/lib/git-core/git }
       /usr/lib/git-core/git:
 
   daemon:
-    # support for git daemon
     essential:
       - git_bins
     contents:
       /usr/lib/git-core/git-daemon:
+      /usr/lib/git-core/git-upload-pack:
```

Comment:
Should this be in the `daemon` slice?

Reply:
This seems to be a leftover from a prior state; the current test runs fine without it. Thanks!